docs/management/admin/event-filters.asciidoc

```diff
@@ -0,0 +1,61 @@
+[[event-filters]]
+[chapter, role="xpack"]
+= Event filters
+
+Event filters allow admins to filter endpoint events that you do not need or want stored in {es} - for example, those that have a high volume or are unhelpful. With event filters, you can optimize your storage in {es}. All endpoint events have the `endpoint.events.network` field.
+
+You can create event filters from the Event filters page or from the Hosts page.
+
+IMPORTANT: Since an event filter blocks it from ever being streamed to {es}, be cognizant if a <<create-custom-rule, custom query rule>> exists that includes any of the same conditions as the event filter, because if so, the rule will not run. This is the expected behavior of event filters.
+
+. To create an event filter via the Hosts page:
+.. Go to *Explore* -> *Hosts*.
+.. Select the *Events* tab to view the Events table.
++
+.. Find the event for which to create a filter, click the *More Actions* button (*...*), then click *Add Endpoint event filter*.
++
+TIP: Since you can only create filters for endpoint events, be sure to filter the Events table to events generated by the Elastic Endpoint. +
+In the KQL search bar, enter the following query: `event.dataset : endpoint.events.network`.
++
+.. Proceed to step 3.
+. To create an event filter via the Event filters page:
+.. Go to *Manage* -> *Event filters*.
+.. Click *Add Event Filter*.
++
+The *Add event filter* flyout opens.
++
+[role="screenshot"]
+image::images/event-filter.png[]
+. Enter a name for the event filter.
+. Depending on from which page you add the filter, either modify the pre-populated conditions that appear, or
+add new conditions that define when {elastic-sec} filters events. You can define multiple conditions with `OR` and `AND` relationships. You can also add nested conditions. In the image above, the event filter filters out events whose `event.category` field is `network`, and whose `process.executable` field is the same as the specified path.
```

Comment:
You can only create Event Filters with `AND` relationships for now.  `OR` will always be disabled